src/Univapay/Enums/QrBrandMerchant.php

```diff
@@ -33,4 +34,13 @@ public static function CHANGIPAY() { return self::create(); }
     public static function HIPAY() { return self::create(); }
     public static function PUBLICBANK() { return self::create('pbengagemy'); }
     public static function MPAY() { return self::create(); }
+    public static function HELLOMONEY() { return self::create(); }
+    public static function TINABA() { return self::create(); }
+    public static function PBENGAGEMY() { return self::create(); }
+    public static function ZALOPAY() { return self::create(); }
+    public static function BLUECODE() { return self::create(); }
+    public static function NAYA_PAY() { return self::create(); }
+    public static function SCB_PLANET_PLUS() { return self::create(); }
+    public static function HUMO() { return self::create(); }
+    public static function MOMO() { return self::create(); }
```

Comment:
We also need to add the following
```suggestion
    public static function MOMO() { return self::create(); }
    public static function NAVERPAY() { return self::create(); }
    public static function TOSSPAY() { return self::create(); }
    public static function OCBC() { return self::create(); }
    public static function HIPAY() { return self::create(); }
    public static function MOMO() { return self::create(); }
```